Update match-phrase-query.asciidoc

damien-renier-elastic opened the pull request:
Added Parameters for consistency

damien-renier-elastic committed:
Update match-phrase-query.asciidoc

Added Parameters for consistency

elasticsearchmachine commented:
@damien-renier-elastic please enable the option "Allow edits and access to secrets by maintainers" on your PR. For more information, [see the documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

damien-renier-elastic committed:
Merge branch '8.17' into damien-renier-elastic-patch-2

elasticsearchmachine commented:
Pinging @elastic/es-docs (Team:Docs)

leemthompo commented:
@damien-renier-elastic the changes show you deleted the paragraph about the `slop`, which doesn't seem to reflect your PR description?

damien-renier-elastic commented:
Removed 2 paragraphes:
- slop
- zero_terms_query

Created the parameters paragraph instead for consistency with other methods.
Added:
- query
- analyzer
- slop
- zero_terms-query

Added a new header for the Analyzer example

leemthompo commented:
@damien-renier-elastic sorry looking at the [changed files](https://github.com/elastic/elasticsearch/pull/118828/files) it still just looks like you _only_ deleted a paragraph about the slop flag

damien-renier-elastic committed:
Update match-phrase-query.asciidoc

damien-renier-elastic commented:
My bad, I somehow managed to reset my modifications after the merge. I double-checked this time.

leemthompo reviewed (approved):
LGTM, thanks!

leemthompo commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|main|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/elasticsearch/121033">](https://github.com/elastic/elasticsearch/pull/121033)|
|✅|8.x|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/elasticsearch/121034">](https://github.com/elastic/elasticsearch/pull/121034)|
|✅|8.16|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/elasticsearch/121035">](https://github.com/elastic/elasticsearch/pull/121035)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

<!--- Backport version: 9.3.0 -->
